zipsplit: add page (#8871)
tldr-pages/tldr: pages/linux/zipsplit.md

# zipsplit

> > Zipsplit reads a zipfile and splits it into smaller zipfiles.

Reviewer: ```suggestion
> Read a zipfile and split it into smaller zipfiles.
```

> > More information: <https://linux.die.net/man/1/zipsplit>.

Reviewer: ```suggestion
> More information: <https://manned.org/zipsplit>.
```
We prefer manned when official documentation isn't available

> - Split zipfile into pieces that are no larger than 1 megabyte:

Reviewer: ```suggestion
- Split zipfile into pieces that are no larger than a particular [n]size:
```

> `zipsplit -n 1048576 archive.zip`

Reviewer: ```suggestion
`zipsplit -n {{size}} {{path/to/archive.zip}}`
```

> - Pause between the creation of each split file:

Reviewer: ```suggestion
- [p]ause between the creation of each split zipfile:
```
We use mnemonics for single-character flags whenever possible.

> `zipsplit -p -n 2097152 archive.zip`

Reviewer: ```suggestion
`zipsplit -p -n {{size}} {{path/to/archive.zip}}`
```

> - Write the split zipfiles into the directory /archive:

Reviewer: ```suggestion
- Output the split zipfiles into the `archive` directory :
```

> `zipsplit -b /archive -n 2097152 archive.zip`

Reviewer: ```suggestion
`zipsplit -b {{archive}} -n {{size}} {{path/to/archive.zip}}`
```
